ENIGMAsystem/SHELL/Platforms/General/POSIX/Utility.cpp

```diff
@@ -103,6 +105,24 @@ unsigned long get_timer() {  // microseconds since the start of the game
   return (enigma::time_current.tv_sec) * 1000000 + (enigma::time_current.tv_nsec / 1000);
 }
 
+std::string shellscript_evaluate(std::string command) {
```

Comment:
You need to declare this in the header if people are going to use it. Also, consider an `execute_shell`-based name for discoverability.

Reply:
well technically evaluate is more of a correct term than execute as those are two different things but yeah i agree that's probably a good idea

Reply:
how's execute_shell_eval()?